src/libraries/Common/src/Interop/OSX/Interop.libc.cs

```diff
@@ -27,5 +27,22 @@ internal struct AttrList
         internal static unsafe partial int setattrlist(string path, AttrList* attrList, void* attrBuf, nint attrBufSize, CULong options);
 
         internal const uint FSOPT_NOFOLLOW = 0x00000001;
+        [LibraryImport(Libraries.libc, EntryPoint = "fsetattrlist", StringMarshalling = StringMarshalling.Utf8, SetLastError = true)]
```

Comment:
```suggestion

        [LibraryImport(Libraries.libc, EntryPoint = "fsetattrlist", StringMarshalling = StringMarshalling.Utf8, SetLastError = true)]
```

Reply:
We typically avoid P/Invoking directly to Unix libraries, instead going through the native shims.  Why is it ok here?

Reply:
Maybe we can defer this to a separate issue, and to someone that has a mac?

Reply:
@tmds Can you create an issue for this so it doesn't fall off the radar?

Reply:
I've created https://github.com/dotnet/runtime/issues/70253.

Reply:
Thank you